tasks/dist/css

```diff
@@ -0,0 +1 @@
+node-sass --output-style expanded --source-map true --precision 6 docs/assets/scss/docs.scss docs/assets/css/docs.css
```

Comment:
Well this whole approach will have issues on Windows. But I'll try it later
or tomorrow to verify this.

On Jul 20, 2016 22:44, "fat" <notifications@github.com> wrote:

> In tasks/dist/css
> <https://github.com/twbs/bootstrap/pull/20332#discussion_r71592896>:
>
> > @@ -0,0 +1 @@
> > +node-sass --output-style expanded --source-map true --precision 6 docs/assets/scss/docs.scss docs/assets/css/docs.css
>
> probably better if you reference the local npm installs of these (in .bin
> ) so that people don't need global installs 👍
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/twbs/bootstrap/pull/20332/files/795f6d924b1c1eb9b637917a41f2c83655408c97#r71592896>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtVO5bBIj0o15D8lDel5ya1-05Smrks5qXnqfgaJpZM4JRHjn>
> .
>